src/api/ledger/parse/orderbook-order.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+const _ = require('lodash');
+const utils = require('./utils');
+const flags = utils.core.Remote.flags.offer;
+const parseAmount = require('./amount');
+
+function parseOrderbookOrder(order: Object): Object {
+  const direction = (order.Flags & flags.Sell) === 0 ? 'buy' : 'sell';
+  const takerGetsAmount = parseAmount(order.TakerGets);
+  const takerPaysAmount = parseAmount(order.TakerPays);
+  const quantity = (direction === 'buy') ? takerPaysAmount : takerGetsAmount;
+  const totalPrice = (direction === 'buy') ? takerGetsAmount : takerPaysAmount;
+
+  const specification = utils.removeUndefined({
+    direction: direction,
+    quantity: quantity,
+    totalPrice: totalPrice,
+    passive: ((order.Flags & flags.Passive) !== 0) || undefined
+  });
+  // "quality" is omitted intentionally as it corresponds to
```

Comment:
Hrmmm.

So there won't be any kind of price per unit of `quantity` property on the order? There may be precision issues but they're possibly inescapable ones, if you need to provide functionality that things like ripple charts need.  I guess you could call it "approx_price" or something to convey the lack of precision. Intentional or not, this seems like a glaring omission.

People would just be left to calculate the price using quantity/totalPrice. How often would be people want a list of orders, and not want to know the quality/price etc? Maybe it should be provided, using the quality where possible.

Interesting related bit of trivia: orders are always executed at the price they were placed at, not what they may end up at after float/rounding issues have potentially caused TakerPays/TakerGets to lose the initial ratio. 

@vhpoet 
@shekenahglory    

Any thoughts? You guys probably have some insight here.